tests/foreman/cli/test_satellitesync.py

```diff
@@ -950,141 +1148,75 @@ def test_negative_reimport_cv_with_same_major_minor(
-                'id': class_export_entities['exporting_cvv_id'],
-            }
-        )
-        exporting_cvv_version = class_export_entities['exporting_cv']['versions'][0]['version']
-        exported_tar = (
-            f'{function_export_cv_directory}/export-{class_export_entities["exporting_cv_name"]}-'
-            f'{exporting_cvv_version}.tar'
-        )
-        result = default_sat.execute(f'[ -f {exported_tar} ]')
-        assert result.status == 0
-        imported_entities = _import_entities(
-            class_export_entities['exporting_prod_name'],
-            class_export_entities['exporting_repo_name'],
-            class_export_entities['exporting_cv_name'],
-        )
-        ContentView.version_import(
-            {
-                'export-tar': exported_tar,
-                'organization-id': imported_entities['importing_org']['id'],
-            }
-        )
+        export_cvv_id = class_export_entities['exporting_cvv_id']
+        export_cv_name = class_export_entities['exporting_cv_name']
+        # Verify export directory is empty
+        assert validate_filepath(default_sat) == ''
+        # Export cv
+        path = ContentExport.completeVersion(
+            {'id': export_cvv_id, 'organization-id': module_org.id}
+        )
+        # grab export dir and check all exported files are there
+        export_dir = os.path.dirname(path['message'])
+        export_folder = os.path.split(export_dir)
+        result = default_sat.execute(f'ls {export_dir}')
+        assert result.stdout != ''
+        # importing portion
+        importing_org = make_org()
+        # set disconnected mode on
+        Settings.set({'name': 'content_disconnected', 'value': "Yes"})
+        default_sat.execute(f'mv {export_dir} {IMPORT_DIR}')
+        import_path = f'{IMPORT_DIR}{export_folder[1]}'
+        default_sat.execute(f'chown -R pulp:pulp {import_path}')
+        # check that files are present in import_path
+        result = default_sat.execute(f'ls {import_path}')
+        assert result.stdout != ''
+        # Import section
+        ContentImport.version({'organization-id': importing_org['id'], 'path': import_path})
         with pytest.raises(CLIReturnCodeError) as error:
-            ContentView.version_import(
-                {
-                    'export-tar': exported_tar,
-                    'organization-id': imported_entities['importing_org']['id'],
-                }
-            )
+            ContentImport.version({'organization-id': importing_org['id'], 'path': import_path})
         assert (
-            f'''the Content View '{class_export_entities["exporting_cv_name"]}' '''
-            'is greater or equal to the version you are trying to import'
+            f'''Content View Version specified in the metadata - '{export_cv_name} 1.0' '''
+            'already exists. If you wish to replace the existing version, '
+            f'''delete {export_cv_name} 1.0 and try again.'''
```

Comment:
This is something leftover from before, and seen a few other times in this file. Mainly an annoyance
```suggestion
            f'Content View Version specified in the metadata - "{export_cv_name} 1.0" '
            'already exists. If you wish to replace the existing version, '
            f'delete {export_cv_name} 1.0 and try again.'
```

Reply:
I kept the single quote for `{export_cv_name} 1.0` to align with the error message so this can pass.

Reply:
ok, then can you instead use double quotes in place of the triple single quotes? :)